docs/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "@xstate/docs",
+  "version": "1.0.0",
+  "description": "Documentation for XState",
+  "main": "index.js",
+  "repository": "https://github.com/davidkpiano/xstate",
+  "author": "David Khourshid",
+  "license": "MIT",
+  "scripts": {
+    "docs:dev": "vuepress dev .",
+    "docs:readme": "cpy ../README.md .",
+    "docs:packages": "cpy '../packages/*/README.md' docs --parents --rename=index.md && cpy '../packages/*/docs/*.md' docs --parents",
+    "docs:build": "npm run docs:readme && npm run docs:packages && vuepress build .",
+    "docs:pdf": "npm run docs:build && vuepress export ."
+  },
+  "dependencies": {
+    "@vuepress/plugin-google-analytics": "^1.8.2",
+    "cpy-cli": "^3.1.1",
+    "gh-pages": "^3.1.0",
```

Comment:
this was probably supposed to be used by the publish script but so far that script is missing

Reply:
Yeah, not sure how to test this... gotta be careful with this.

Reply:
Well, you'll have to do a test run on production :trollface: Unless you want to go through the trouble of testing this on a fork.